src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexWriteOperation.cs

```diff
@@ -52,6 +49,17 @@ public CoraxIndexWriteOperation(Index index, Transaction writeTransaction, Corax
             {
                 throw new IndexWriteException(e);
             }
+            
+            if (index.Definition.HasDynamicFields)
+            {
+                _dynamicFieldsBuilder = new IndexFieldsMappingBuilder(true, true)
```

Comment:
See my above comment - here you could have:

`IndexFieldsMappingBuilder.CreateForWriter(supportsDynamicFields: true)`